src/schema/v2/artist/index.ts

```diff
@@ -240,6 +240,10 @@ export const ArtistType = new GraphQLObjectType<any, ResolverContext>({
           )
         },
       },
+      awards: {
+        type: GraphQLNonNull(GraphQLString),
+        resolve: ({ awards }) => awards || "",
+      },
```

Comment:
Why did you choose make this type `GraphQLNonNull`?

Reply:
I believe it's a safe pattern to fallback to an empty string because this field is of type string in Gravity. We still have some records that are `nil`, so using an empty string gives should us consistency.

Here is the [discussion](https://github.com/artsy/metaphysics/pull/5564#discussion_r1484024204) with @dzucconi - it might help 😅

Reply:
Actually, this should probably be nullable.

Integers and Booleans should _never_ be nullable because the null value is essentially either `false` or `0`; in this case empty string is useless.